cmd/kpod/save.go

```diff
@@ -81,9 +87,15 @@ func saveCmd(c *cli.Context) error {
 // saveImage pushes the image to docker-archive or oci by
 // calling pushImage
 func saveImage(store storage.Store, opts saveOptions) error {
-	dst := dockerArchive + opts.output
+	var dst string
+	switch opts.format {
+	case "oci":
+		dst = ociArchive + opts.output
+	default:
+		dst = dockerArchive + opts.output
```

Comment:
I think it would be much safer for users to only explicitly allow the expected values (`"docker-archive"` and `""`, perhaps) and fail on unknown values of `opts.format`; both to protect against typos, and to cleanly fail if some future version of the tool introduces other formats and the user tries to use the new formats with an old version of the tool.

Reply:
Fixed.

Reply:
AFAICS the `--format` option is now mandatory; is that intentional? (I’m not saying it can’t be mandatory; I just want to make sure this has been a conscious decision.)